lib/game.rb

```diff
@@ -30,17 +30,17 @@ def self.lowest_total_score
 
   def self.percentage_home_wins
     home_wins = @@all.find_all { |game| game.home_goals > game.away_goals}.count
-    ( home_wins.to_f / @@all.length.to_f ).round(1) * 100
+    ( home_wins.to_f / @@all.length.to_f ).round(2)
   end
 
   def self.percentage_visitor_wins
     visitor_wins = @@all.find_all { |game| game.home_goals < game.away_goals}.count
-    ( visitor_wins.to_f / @@all.length.to_f ).round(1) * 100
+    ( visitor_wins.to_f / @@all.length.to_f ).round(2)
   end
 
   def self.percentage_ties
     ties = @@all.find_all { |game| game.home_goals == game.away_goals}.count
-    ( ties.to_f / @@all.length.to_f ).round(1) * 100
+    ( ties.to_f / @@all.length.to_f ).round(2)
```

Comment:
Nice, this will fix our RSpec errors for the percentages being displayed incorrectly.